joomla/joomla-cms
# [4.0] Namespace com_categories

### Summary of Changes
This simple PR convert com_categories component to namespace MVC for Joomla 4.

### Testing Instructions
Overall, test to make sure it works in the same as before (none namespace version)

1. Install latest Joomla 4.0-dev with sample data (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_categories.zip)

2. Login to administrator area of the site, access to Content -> Categories.

3. Try to add/edit/publish/unpublish/archieve categories, make sure it is working as expected

### Additional comment
com_categories require different access permission check compare to most backend component. I ignore the check for now and wait for decision about the method I propose on PR https://github.com/joomla/joomla-cms/pull/15812 . If it is accepted, I will implement the check here or on a separate PR.
